utils/hct/hctdb.py

```diff
@@ -1559,6 +1542,8 @@ def build_valrules(self):
         self.add_valrule("Meta.ForceCaseOnSwitch", "Attribute forcecase only works for switch")
         self.add_valrule("Meta.ControlFlowHintNotOnControlFlow", "Control flow hint only works on control flow inst")
         self.add_valrule("Meta.TextureType", "elements of typed buffers and textures must fit in four 32-bit quantities")
+        self.add_valrule("Meta.BarycentricsInterpolation", "Invalid interpolation type '%0' for SV_Barycentrics. Interpolation type must be linear, linear noperspective, linear centroid, linear noperspective centroid, linear sample or linear noperspective sample")
```

Comment:
This should use the add_valrule_msg function with separate summary line and message with %0 value.  Also, since the only invalid enum here is Constant (nointerpolation), why not shorten the message to something like "SV_Barycentrics can not be used with interpolation type 'nointerpolation'" or something like that.